src/ios/MusicControls.m

```diff
@@ -171,17 +174,68 @@ - (void) handleMusicControlsNotification: (NSNotification *) notification {
     }
 }
 
+- (void) nextTrackEvent:(MPRemoteCommandEvent *)event {
+    NSString * action = @"music-controls-next";
+    NSString * jsonAction = [NSString stringWithFormat:@"{\"message\":\"%@\"}", action];
+    CDVPluginResult * pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_OK messageAsString:jsonAction];
+    [self.commandDelegate sendPluginResult:pluginResult callbackId:[self latestEventCallbackId]];
+}
+- (void) prevTrackEvent:(MPRemoteCommandEvent *)event {
+    NSString * action = @"music-controls-previous";
+    NSString * jsonAction = [NSString stringWithFormat:@"{\"message\":\"%@\"}", action];
+    CDVPluginResult * pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_OK messageAsString:jsonAction];
+    [self.commandDelegate sendPluginResult:pluginResult callbackId:[self latestEventCallbackId]];
+}
+
+- (void) skipForwardEvent:(MPSkipIntervalCommandEvent *)event {
+    NSString * action = @"music-controls-skip-forward";
+    NSString * jsonAction = [NSString stringWithFormat:@"{\"message\":\"%@\"}", action];
+    CDVPluginResult * pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_OK messageAsString:jsonAction];
+    [self.commandDelegate sendPluginResult:pluginResult callbackId:[self latestEventCallbackId]];
+}
+
+- (void) skipBackwardEvent:(MPSkipIntervalCommandEvent *)event {
+    NSString * action = @"music-controls-skip-backward";
+    NSString * jsonAction = [NSString stringWithFormat:@"{\"message\":\"%@\"}", action];
+    CDVPluginResult * pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_OK messageAsString:jsonAction];
+    [self.commandDelegate sendPluginResult:pluginResult callbackId:[self latestEventCallbackId]];
+}
+
 - (void) registerMusicControlsEventListener {
     [[UIApplication sharedApplication] beginReceivingRemoteControlEvents];
     [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(handleMusicControlsNotification:) name:@"musicControlsEventNotification" object:nil];
 
-    // NSFoundationVersionNumber_iOS_9_0 = 1240.100000
-
-    if (floor(NSFoundationVersionNumber) > 1240.100000) {
+    if (floor(NSFoundationVersionNumber) > NSFoundationVersionNumber_iOS_9_0) {
       //only available in iOS 9.1 and up.
       MPRemoteCommandCenter *commandCenter = [MPRemoteCommandCenter sharedCommandCenter];
       [commandCenter.changePlaybackPositionCommand setEnabled:true];
       [commandCenter.changePlaybackPositionCommand addTarget:self action:@selector(changedThumbSliderOnLockScreen:)];
+
+      if (musicControlsSettings.hasNext) {
+        MPRemoteCommand *nextTrackCommand = [commandCenter nextTrackCommand];
+        [nextTrackCommand setEnabled:YES];
+        [nextTrackCommand addTarget:self action:@selector(nextTrackEvent:)];
```

Comment:
Indentation here is 2 spaces, let's use 4 so it's consistent with the rest of the code